### PR TITLE
Updated NL Strings

### PR DESCRIPTION
**Pull request by Pandemic-XDA**
add some new lines for icon app option

### CODE REVIEW

**Review by yujincheng08 — APPROVED** [review]
thx

**Comment by yujincheng08** [comment]
> ERROR:/home/runner/work/LSPosed/LSPosed/app/src/main/res/values-nl/strings.xml: Resource and asset merger: Found item String/settings_show_hidden_icon_apps_enabled more than one time

**Comment by Pandemic-XDA** [comment]
> > ERROR:/home/runner/work/LSPosed/LSPosed/app/src/main/res/values-nl/strings.xml: Resource and asset merger: Found item String/settings_show_hidden_icon_apps_enabled more than one time

will fix that

**Comment by Pandemic-XDA** [comment]
> > ERROR:/home/runner/work/LSPosed/LSPosed/app/src/main/res/values-nl/strings.xml: Resource and asset merger: Found item String/settings_show_hidden_icon_apps_enabled more than one time

which line need to edit ?

File: app/src/main/res/values-nl/strings.xml [hunk 1]
```diff
@@ -172,5 +172,9 @@
     <string name="verbose_log_not_avaliable">Uitgebreid logboek is niet beschikbaar. Als u het net hebt ingeschakeld zonder opnieuw op te starten, probeer het dan opnieuw na het opnieuw opstarten.</string>
     <string name="NotInstall">Niet geïnstalleerd</string>
     <string name="NotInstallDetail">LSPosed is niet geïnstalleerd</string>
-    <string name="translators"><![CDATA[<a href="https://github.com/Pandemic-XDA">Pandemic (Pandemic-XDA)</a>]]></string>
+    <string name="settings_show_hidden_icon_apps_enabled">(Android 10+) Beperk apps om hun opstartpictogrammen te verbergen</string>
+    <string name="settings_show_hidden_icon_apps_enabled">Forceer apps om launcher-pictogrammen weer te geven</string>
```
Comment (yujincheng08) [inline app/src/main/res/values-nl/strings.xml:176]: The name of this line is wrong.

File: app/src/main/res/values-nl/strings.xml [hunk 2]
```diff
@@ -172,5 +172,9 @@
...
+    <string name="settings_show_hidden_icon_apps_enabled_summary">Na Android 10 mogen apps (vooral Xposed-modules) hun opstartpictogrammen niet verbergen. Schakel de schakelaar uit om deze functie uit te schakelen.</string>
```
Comment (yujincheng08) [inline app/src/main/res/values-nl/strings.xml:177]: this too

**Review by yujincheng08 — CHANGES_REQUESTED** [review]

File: app/src/main/res/values-nl/strings.xml (hunk 1 above)
Comment (Pandemic-XDA) [reply app/src/main/res/values-nl/strings.xml:176]: yes saw it thanks mate, my bad :)

Comment (Pandemic-XDA) [reply app/src/main/res/values-nl/strings.xml:176]: but any advice to rename it into what ?

**Review by Pandemic-XDA — COMMENTED** [review]
fixed

**Comment by yujincheng08** [comment]
seriously?

**Comment by Pandemic-XDA** [comment]
> seriously?

yes see or did i something wrong ?

**Comment by yujincheng08** [comment]
> > seriously?
> 
> yes see or did i something wrong ?

you did not translate `system_inject_fail_summary` and `system_inject_fail`

**Comment by Pandemic-XDA** [comment]
> > > seriously?
> > 
> > 
> > yes see or did i something wrong ?
> 
> you did not translate `system_inject_fail_summary` and `system_inject_fail`

where the hell i find those lines then ?

**Comment by yujincheng08** [comment]
> > seriously?
> 
> yes see or did i something wrong ?

https://github.com/LSPosed/LSPosed/blob/fd32c3107ab7ba43ce6ab9ac899859cb76dc1257/app/src/main/res/values/strings.xml#L44-L45

**Comment by Pandemic-XDA** [comment]
> > > seriously?
> > 
> > 
> > yes see or did i something wrong ?
> 
> https://github.com/LSPosed/LSPosed/blob/fd32c3107ab7ba43ce6ab9ac899859cb76dc1257/app/src/main/res/values/strings.xml#L44-L45

oke thanks, maybe its usefull to tell all the helpers which lines are added to translate.

**Comment by yujincheng08** [comment]
> > > > seriously?
> > > 
> > > 
> > > yes see or did i something wrong ?
> > 
> > 
> > https://github.com/LSPosed/LSPosed/blob/fd32c3107ab7ba43ce6ab9ac899859cb76dc1257/app/src/main/res/values/strings.xml#L44-L45
> 
> oke thanks, maybe its usefull to tell all the helpers which lines are added to translate.

This is helpful

https://github.com/LSPosed/LSPosed/commits/master/app/src/main/res/values/strings.xml

**Comment by Pandemic-XDA** [comment]
done

**Comment by yujincheng08** [comment]
still lacks many strings. Plz refer to this PR and make sure you translate all the updated strings: #630

**Comment by Pandemic-XDA** [comment]
> still lacks many strings. Plz refer to this PR and make sure you translate all the updated strings: #630

how do you mean to many ? it just change all the time and can you provide me a english strings.xml ? correct one ? then i start all over again.....

**Review by yujincheng08 — APPROVED** [review]

**Pull request merged**